src/metabase/api/transform.clj

```diff
@@ -0,0 +1,15 @@
+(ns metabase.api.transform
+  (:require [compojure.core :refer [GET]]
+            [medley.core :as m]
+            [metabase.api.common :as api]
+            [metabase.transforms
+             [core :as transform]
+             [specs :as transform.specs]]))
+
+(api/defendpoint GET "/:db-id/:schema/:transform-name"
+  [db-id schema transform-name]
+  (->> @transform.specs/transform-specs
```

Comment:
Looks like there aren't any perms checks here. What perms are needed to use this endpoint?